SKYKitChat.podspec

```diff
@@ -14,14 +14,19 @@ This is the client library for the Skygear Chat extension.
 
   s.ios.deployment_target = '8.0'
 
-  s.source_files = 'SKYKitChat/Classes/**/*'
-  s.dependency 'SKYKit', '~> 0.19.0'
-  
-  # s.resource_bundles = {
-  #   'SKYKitChat' => ['SKYKitChat/Assets/*.png']
-  # }
-
-  # s.public_header_files = 'Pod/Classes/**/*.h'
-  # s.frameworks = 'UIKit', 'MapKit'
-  # s.dependency 'AFNetworking', '~> 2.3'
+  s.subspec 'Core' do |sp|
+    sp.source_files = 'SKYKitChat/Classes/Core/**/*'
```

Comment:
There are nested directories originally in `SKYKitChat/Classes`. And I think it is better to separate the source code in `Core` subspec and `UI` subspec. That is why I move the original source code into one directory deeper.